ui/app/components/totp-edit.hbs

```diff
@@ -26,17 +26,18 @@
 </PageHeader>
 
 {{#if (eq @mode "show")}}
-  <Totp::KeyDetails @model={{@model}} @delete={{this.delete}} />
+  <Totp::KeyDetails @model={{@model}} @delete={{this.deleteKey}} />
 {{else if (eq @mode "create")}}
   {{#if this.hasGenerated}}
     <Totp::KeyQrCode @model={{@model}} @reset={{this.reset}} />
   {{else}}
     <Totp::KeyCreate
-      @create={{this.create}}
+      @create={{this.createKey}}
```

Comment:
Nice job! This is much clearer, I'd go a step further and make the arg more explicit here, too
```suggestion
      @onSubmit={{this.createKey}}
```